lang/tags/functions.py

```diff
@@ -11,6 +11,12 @@
 mod.list(
     "code_parameter_name", desc="List of common parameter names for active language"
 )
+mod.list(
+    "code_function_modifiers",
```

Comment:
Prefer singular for talon list names

```suggestion
    "code_function_modifier",
```